### PR TITLE
remove unnecessary params

### PR DESCRIPTION
**Pull request by liyaqin1**

### CODE REVIEW

File: deploy-board/deploy_board/templates/clusters/cluster_configuration.html [hunk 1]
```diff
@@ -725,7 +725,7 @@ <h4 class="panel-title pull-left pointer-cursor">
                     var provider = capacitySetting.currentProvider;
                     var cell = capacitySetting.currentCell;
                     var arch = value;
-                    value = '{{ default_cmp_image }}';
+                    //value = '{{ default_cmp_image }}';
```
Comment (yongwen) [inline deploy-board/deploy_board/templates/clusters/cluster_configuration.html:728]: will this remove the default image in UI?

File: deploy-board/deploy_board/templates/clusters/cluster_configuration.html [hunk 2]
```diff
@@ -789,6 +789,7 @@ <h4 class="panel-title pull-left pointer-cursor">
                 imageNameChange: function(value) {
                     var cell = capacitySetting.currentCell;
                     capacitySetting.imageNameValue = value;
+                    console.log("image name:", value);
```
Comment (yongwen) [inline deploy-board/deploy_board/templates/clusters/cluster_configuration.html:792]: can we remove this debug log?

Comment (liyaqin1) [reply deploy-board/deploy_board/templates/clusters/cluster_configuration.html:792]: sure

File: deploy-board/deploy_board/templates/clusters/cluster_configuration.html [hunk 3]
```diff
@@ -744,12 +744,12 @@ <h4 class="panel-title pull-left pointer-cursor">
                                     function (o) {
                                         return {
                                             value: o,
-                                            text: o, isSelected: o === value
+                                            text: o, isSelected: o === data[0]
```
Comment (tylerwowen) [inline deploy-board/deploy_board/templates/clusters/cluster_configuration.html:747]: Maybe
```
let imageName = data[0];
...
text: o, isSelected: o === imageName
...
```

**Review by tylerwowen — APPROVED** [review]

**Pull request merged**